ios-contact-manager/ios-contact-manager/Controller/EditViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  EditViewController.swift
+//  ios-contact-manager
+//
+//  Created by Swain Yun on 10/10/23.
+//
+
+import UIKit
+
+final class EditViewController: UIViewController {
+    @IBOutlet private weak var nameTextField: UITextField!
+    @IBOutlet private weak var ageTextField: UITextField!
+    @IBOutlet private weak var phoneNumberTextField: UITextField!
+    
+    weak var delegate: ContactsManagable?
+    var contact: Contact?
```

Comment:
```suggestion
    private var contact: Contact?
    
    init(contact: Contact) 
```

init 을 통해 모델값을 주입해보도록 해보세요. 스토리보드를 통해 뷰컨트롤러를 만들었어도 주입이 가능합니다 🙂